### ISSUE TITLE
Pico onboard LED support 

### ISSUE THREAD

**Issue by tokiAi** [adafruit/u2if#13]
Hello,

I am using a Raspberry Pi Pico, and I noticed that the onboard LED is typically referenced as GPIO 25 or board.LED. However, I am unsure if it is fully supported in the current version of the u2if firmware.

Could you please clarify:

    Is the onboard LED on GPIO 25 (or board.LED) officially supported for the Pico board?
    If so, are there any specific configurations or initialization steps required to enable or use it?
    If not currently supported, are there any plans to add support in future releases?

Additionally, this feature is nice to have for testing the basic LED blink script after setting up `u2if`, as it would allow a quick check that everything is working as expected.

Thank you for your time and assistance!

**Comment by caternuson** [comment]
The u2if firmware probably supports this. Access to it would depends on how you are interfacing with the Pico running the u2if firmware.

For example, Blinka does **not** provide access to GP25:
https://github.com/adafruit/Adafruit_Blinka/blob/main/src/adafruit_blinka/board/pico_u2if.py

**Comment by freemansoft** [comment]
Yes, I noticed this also.  Was wondering if this was so that the same U2IF could be used for Pico and Pico W?  The LED on the Pico W is not attached to pin 25 and pin25 is tied up communicating with the wireless
